react/src/components/calculators/late-refunds/LateRefundsForm.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useContext} from 'react'
+
+// components
+import TextInput from '../../helpers/formhelpers/TextInput'
+import LateRefundsTable from './LateRefundsTable'
+
+// types
+import {LateRefundsContext} from './LateRefundsContext'
+import InterestRatesTable from '../shared/InterestRatesTable'
+
+function LateRefundsForm() {
+  const {
+    bankHolidaysNo,
+    setBankHolidaysNo,
+    errors,
+    rates
+  } = useContext(LateRefundsContext)
+
+  const handleChange = (e:  React.ChangeEvent<HTMLInputElement>) => {
+    setBankHolidaysNo(e.currentTarget.value)
+  }
+
+  return (
+    <>
+      <div className="govuk-form-group">
+        <TextInput
+          labelText="Number of bank holidays (optional)"
+          name="bankHolidays"
+          inputClassName="govuk-input govuk-input--width-3"
+          inputValue={bankHolidaysNo}
+          error={errors.bankHolidays}
+          onChangeCallback={(e) => handleChange?.(e)}

```

Comment:
this could just be `onChangeCallback={handleChange}` given it's defined as a `const` within this component